internal/pkg/cli/pipeline_init.go

```diff
@@ -574,17 +574,17 @@ func (o *initPipelineOpts) createPipelineManifest() error {
 	}
 
 	var manifestExists bool
-	manifestPath, err := o.workspace.WritePipelineManifest(manifest, o.name)
+	o.manifestPath, err = o.workspace.WritePipelineManifest(manifest, o.name)
 	if err != nil {
 		e, ok := err.(*workspace.ErrFileExists)
 		if !ok {
 			return fmt.Errorf("write pipeline manifest to workspace: %w", err)
 		}
 		manifestExists = true
-		manifestPath = e.FileName
+		o.manifestPath = e.FileName
 	}
 
-	manifestPath, err = relPath(manifestPath)
+	o.manifestPath, err = relPath(o.manifestPath)
```

Comment:
Do we need to change here as well?

Reply:
I took away `manifestPath` altogether.

Reply:
Im still not clear why we don't need `ws.Rel` here. Is it because we require them to run `pipeline_init` within a workspace? But what if they are running the command within a sub dir?

Reply:
Fixed. Thanks!